src/com/lilithsthrone/game/character/GameCharacter.java

```diff
@@ -11483,12 +11483,7 @@ public String incrementHealth(GameCharacter attacker, float increment) {
 							+ " <b>"+(manaLoss)+"</b>"+ " <b style='color:" + Attribute.DAMAGE_LUST.getColour().toWebHexString() + ";'>lust damage</b> as [npc.she] [npc.verb(get)] aroused by inflicting damage!"
 						+ "</p>"))
 						+incrementLust(manaLoss, false);
-				
-				
-				
-			} else {
```

Comment:
this is still needed

Reply:
not if we also get rid of the following else statement, which i forgot to do. then we'd just let it fall through to the final return?